src/core/solvers/ddp.cpp

```diff
@@ -369,6 +381,9 @@ void SolverDDP::allocateData() {
   const std::size_t ndx = problem_->get_ndx();
   const std::size_t nu = problem_->get_nu_max();
   const std::vector<boost::shared_ptr<ActionModelAbstract> >& models = problem_->get_runningModels();
+#ifdef CROCODDYL_WITH_MULTITHREADING
+#pragma omp parallel for num_threads(NUM_THREADS)
```

Comment:
This here allocates a lot more than the two cases above, so there might be a benefit. I guess there is?

Reply:
Since there is no additional computational benefit, I think we can keep the memory allocation without multithreading everywhere.